charts/library/common/Chart.yaml

```diff
@@ -2,8 +2,8 @@ apiVersion: v2
 name: common
 description: Function library for Helm charts
 type: library
-version: 0.3.0
-kubeVersion: ">=1.16.0-0"
+version: 0.4.0
```

Comment:
The `development` branch version has already been bumped, shouldn't require another version bump

Reply:
Weirdly enough I can't find an exact minimum version that the Gateway API needs, all I found is that they want to support the latest 5 minor versions. Personally I've been running it on 1.22.6 which is much newer than this, but I'll try to find something out about this.

Reply:
Alright, with the branch being reopened, is this still the case?